core/Tracker/VisitExcluded.php

```diff
@@ -89,8 +89,11 @@ public function isExcluded()
          * @param bool &$excluded Whether the request should be excluded or not. Initialized
          *                        to `false`. Event subscribers should set it to `true` in
          *                        order to exclude the request.
+         * @param Request $request The request object which contains all of the request's information
+         * @param bool|string $userAgent The user-agent used by the visitor
+         * @param bool|string $ip The IP-address of the visitor
          */
-        Piwik::postEvent('Tracker.isExcludedVisit', array(&$excluded));
+        Piwik::postEvent('Tracker.isExcludedVisit', array(&$excluded, $this->request, $this->userAgent, $this->ip));
```

Comment:
This is a bit confusing here because the `$this->request` contains a method to get the userAgent and the IP as well. So can maybe someone explain (and afterwards add to documentation) when the passed userAgent and the IP is different to the one in the request?